File: task1.swift
print("Enter the string: ")
let stroka = readLine()!
let vowels = "aeyuoiAEYUOI"
let consonants = "bcdfghjklmnpqrstvwxzBCDFGHJKLMNPQRSTVWXZ"
var num_vowels = 0
var num_consonants = 0

for i in stroka {
    if vowels.contains(i) {
        num_vowels += 1
    }
}

for j in stroka {
    if consonants.contains(j) {
        num_consonants += 1
    }
}

if num_vowels > num_consonants {
    print("Yes")
} else if num_vowels < num_consonants {
    print("No")
} else {
    print("Identical")
}

File: task1.cs
using System;

class Program
{
    static void Main()
    {
        Console.Write("Enter the string: ");
        string stroka = Console.ReadLine();
        string vowels = "aeioyuAEIOYU";
        string consonants = "bcdfghjklmnpqrstvwxzBCDFGHJKLMNPQRSTVWXZ";
        int num_vowels = 0;
        int num_consonants = 0;

        foreach (char c in stroka)
        {
            if (vowels.Contains(c))
            {
                num_vowels++;
            }
          
        }
        foreach (char c in stroka)
        {
            if (consonants.Contains(c))
            {
                num_consonants++;
            }
          
        }

        if (num_vowels > num_consonants)
        {
            Console.WriteLine("Yes");
        }
        else if (num_vowels < num_consonants)
        {
            Console.WriteLine("No");
        }
        else
        {
            Console.WriteLine("Identical");
        }
    }
}
